docs/sources/enterprise/access-control/permissions.md

```diff
@@ -21,55 +21,57 @@ scope
+reports.settings:write | n/a | Update report settings.
+reports.settings:read | n/a | Read report settings.
+provisioning:reload | service:access-control | Reload provisioning files.
+users:read | global:users:* | Read or search user profiles.
+users:write | global:users:* | Update a user’s profile.
+users.teams:read | global:users:* | Read a user’s teams.
+users.authtoken:list | global:users:* | List authentication tokens that are assigned to a user.
+users.authtoken:update | global:users:* | Update authentication tokens that are assigned to a user.
+users.password:update | global:users:* | Update a user’s password.
+users:delete | global:users:* | Delete a user.
+users:create | n/a | Create a user.
+users:enable | global:users:* | Enable a user.
+users:disable | global:users:* | Disable a user.
+users.permissions:update | global:users:* | Update a user’s organization-level permissions.
+users:logout | global:users:* | Log out a user.
+users.quotas:list | global:users:* | List a user’s quotas.
+users.quotas:update | global:users:* | Update a user’s quotas.
+org.users.read | users:* | Get user profiles within an organization.
+org.users.add | users:* | Add a user to an organization.
+org.users.remove | users:* | Remove a user from an organization.
+org.users.role:update | users:* | Update the organization role (`Viewer`, `Editor`, `Admin`) for an organization.
+ldap.user:read | n/a | Get a user via LDAP.
+ldap.user:sync | n/a | Sync a user via LDAP.
+ldap.status:read | n/a | Verify the LDAP servers’ availability.
+status:accesscontrol | service:access-control | Get access-control enabled status.
 
 ## Scope definitions
 
-Note that below list is not exhaustive yet and more scopes will be available with further releases of fine-grained access control.
+The following list contains fine-grained access control scopes.
 
-Scope | Description
---- | --- 
-roles:* | Indicates against what roles an action can be performed. For example, `roles:*` assumes any roles, and `roles:randomuid` assumes only a role with UID `randomuid`.  
-permissions:delegate | The scope is only applicable for roles associated with the Access Control itself and indicates that you can delegate your permissions only, or a subset of it, by creating a new role or making an assignment. 
-reports:* | Indicates against what reports an action can be performed.
-service:access-control | Only relevant for provisioning and indicates that the action can be performed only for access control provisioning files.
-global:users:* | Indicates that action can be performed against users globally.
-users:* | Indicates that an action can be performed against users in organization level.
+Scopes | Descriptions
+--- | ---
+roles:* | Restrict an action to a set of roles. For example, `roles:*` matches any role, `roles:randomuid` matches only the role with UID `randomuid` and `roles:custom:reports:{editor,viewer}` matches both `custom:reports:editor` and `custom:reports:viewer` roles.
```

Comment:
Those are not valid UIDs post https://github.com/grafana/grafana-enterprise/issues/1466

Reply:
Good point. We did not backport that change. But if we still need to change this, then conf/access-control/sample.yaml and docs/sources/enterprise/access-control/provisioning.md need to be updated

Reply:
the {editor,viewer} ?

Reply:
I'd recommend to keep it simple for now and give an example of restricting to one role UID.

Reply:
Colons are not valid characters in UIDs post URI restrictions `:`. Good point about provisioning docs and samples, we'll have to fix that.